docs/hazmat/primitives/asymmetric/serialization.rst

```diff
@@ -226,9 +226,45 @@ all begin with ``-----BEGIN {format}-----`` and end with ``-----END
     :raises cryptography.exceptions.UnsupportedAlgorithm: If the serialized parameters
         is of a type that is not supported by the backend.
 
+PKCS12
+~~~~~~
+
+.. currentmodule:: cryptography.hazmat.primitives.serialization.pkcs12
+
+PKCS12 is a binary format described in :rfc:`7292`. It can contain
+certificates, keys, and more. PKCS12 files commonly have a ``pfx`` or ``p12``
+file suffix.
+
+.. note::
+
+    ``cryptography`` only supports a single private key and associated
+    certificates when parsing PKCS12 files at this time.
+
+.. function:: load_key_and_certificates(data, password, backend)
+
+    .. versionadded:: 2.5
+
+    Deserialize a PKCS12 blob.
+
+    :param bytes data: The binary data.
+
+    :param bytes password: The password to use to decrypt the data. ``None``
+        if the PKCS12 is not encrypted.
+
+    :param backend: A backend instance.
+
+    :returns: A tuple of
+        ``(private_key, certificate, [additional_certificates])``.
+        ``private_key`` is a private key type or ``None``, ``certificate``
+        is either a :class:`~cryptography.x509.Certificate` or ``None``, and
+        ``additional_certificates`` is a list of
+        :class:`~cryptography.x509.Certificate` instances.
```

Comment:
Please explain the difference between the first certificate and the additional ones.